src/søknad/steg/4-barnasbosted/BarnetsBostedEndre.tsx

```diff
@@ -30,6 +26,23 @@ import { EBorAnnenForelderISammeHus } from '../../../models/steg/barnasbosted';
 import SeksjonGruppe from '../../../components/gruppe/SeksjonGruppe';
 import BarnetsAndreForelderTittel from './BarnetsAndreForelderTittel';
 import LocaleTekst from '../../../language/LocaleTekst';
+import { erForelderUtfylt } from '../../../overgangsstønad/steg/5-aktivitet/helper';
+
+const lagOppdatertBarneliste = (
+  barneliste: IBarn[],
+  nåværendeBarn: IBarn,
+  forelder: IForelder
+) => {
+  return barneliste.map((b) => {
+    if (b === nåværendeBarn) {
+      let nyttBarn = nåværendeBarn;
+      nyttBarn.forelder = forelder;
```

Comment:
Her tror jeg du også oppdaterer `nåværendeBarn`, siden både nyttBarn og nåværendeBarn peker på samme objekt. Hvis det ikke har noe å si at den originale barnelisten blir endret, så går dette fint, ellers kan man bruke `cloneDeep` fra lodash.

Reply:
Mm, jeg har egentlig bare trukket ut koden som lå direkte leggTilForelder()-metoden i en egen metode siden den brukes to ganger. HAr ikke lagt merke til noe annerledes, så tenker at det går greit :P

Reply:
Okay, da bør det fortsatt funke ja 😄